{"name": "Проверка получения ошибки аутентификации курьера при вводе невалидных данных", "status": "passed", "description": "В тест по очереди передаются наборы данных с несуществующим логином или неверным паролем. Проверяются код и тело ответа.", "parameters": [{"name": "nonexistent_credentials", "value": "{'login': 'Igor12', 'password': '1233'}"}], "start": 1726902693619, "stop": 1726902694415, "uuid": "13a07b3a-118d-4e16-ac27-67bcf92f2848", "historyId": "43d00e36908b52ae9ba9f4b847c22533", "testCaseId": "199be2f2af7455b14ed4795a4ee970da", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_nonexistent_data_not_found", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-I4FH3MC"}, {"name": "thread", "value": "105144-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}